src/ApplicationInsights.Kubernetes.csproj

```diff
@@ -51,15 +51,18 @@
 
   <ItemGroup Condition="'$(TargetFramework)'=='netstandard1.3'">
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="1.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.2" />
```

Comment:
If you depend on Ms.Ex.Logging, do you still need Logging.Abstractions?

Reply:
@karolz-ms good eye 👍 I'll fix it before the merge.